coremltools/converters/mil/frontend/torch/utils.py

```diff
@@ -0,0 +1,10 @@
+#  Copyright (c) 2023, Apple Inc. All rights reserved.
+#
```

Comment:
Do we really need this extra file?
Can't we just put the class into either `op.py` or `converer.py`?

Reply:
Putting in an extra file rather is to avoid "circular import": e.g., `ops.py` needs this `TorchFrontend`, but `ops.py` cannot `import .converter` since `converter.py` also does `import .ops`

Reply:
Yeah, so why not defining this in `ops.py`?
`converter.py` can still import this class from `.ops`?

Reply:
Make sense, `ops.py` is the lowest level so should not get circular import (at least for now)